recipes/mavsdk/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,8 @@
+#include <mavsdk.h>
+
+int main()
+{
+    mavsdk::Mavsdk mavsdk;
+    std::string version = mavsdk.version();
```

Comment:
```suggestion
#include <mavsdk.h>
#include <iostream>

int main()
{
    mavsdk::Mavsdk mavsdk;
    std::string version = mavsdk.version();
    std::cout << "MAVSDK version: " << version << std::endl;
```

To ensure the compiler does not optimize these calls away